### ISSUE TITLE
intersight_iam_end_point_user data source does not allow specifying organization

### ISSUE THREAD

**Issue by dchosnek** [CiscoDevNet/terraform-provider-intersight#61]
In 1.0.0 and 1.0.1 I could specify the organization for an end point user data source. In 1.0.2, I get an error

`Error: rpc error: code = Unavailable desc = transport is closing.`

This works in 1.0.0 and 1.0.1 but fails in 1.0.2 unless I remove the organization block:

```
data "intersight_iam_end_point_user" "existing_admin" {
  name = "admin"
  organization {
    moid = data.intersight_organization_organization.default_organization.moid
  }
}
```

**Comment by code-lucidal58** [comment]
Data sources with simple type properties only are supported today. We might add this in future but there are no roadmaps for the same. For now, labelling this issue as `wontfix`.

**Comment by code-lucidal58** [comment]
Issue fixed in 1.0.15 release
